types/openapi-factory/openapi-factory-tests.ts

```diff
@@ -6,24 +6,29 @@ api.setAuthorizer((req: any) => req);
 api.onEvent((req: any) => req);
 api.onSchedule((req: any) => req);
 
-const testReponse = {
-    statusCode: 200
-};
+/* tslint:disable-next-line:no-unnecessary-generics */
+function getResponse<T>(request: OpenApiFactory.HttpRequest<T>) {
```

Comment:
Here, your generic `T` isn't helping you at all, since you can't depend on anything being true about it in the body of the function.